README.rst

```diff
@@ -65,6 +65,18 @@ gcm
   to contain the 'Server key', which can be acquired from Firebase Console at:
   ``https://console.firebase.google.com/project/<PROJECT NAME>/settings/cloudmessaging/``
 
+Pusher `data` configuration
+=======
+
+The following parameters can be specified in the `data` dictionary which is given when configuring the pusher
+via [POST /_matrix/client/r0/pushers/set](<https://matrix.org/docs/spec/client_server/latest#post-matrix-client-r0-pushers-set>) :
```

Comment:
this is RST, so this format doesn't work. I think I got it right in my suggestion?

Reply:
Ah sorry yes, you're right. I was bitten by my previewer. Updated.